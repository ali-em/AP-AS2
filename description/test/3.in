4 4
--o-
oooo
o-#-
o---
