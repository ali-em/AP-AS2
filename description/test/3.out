Insane!
1 3 D
2 1 R
2 4 L
4 1 U
2 1 R
3 3 U
